### ISSUE TITLE
.incr() docs incorret or unclear

### ISSUE THREAD

**Issue by wesbos** [smallwins/docs.begin.com#93]
Just trying to increment a value and the docs example didn't work, things that should be noted:

1. The property must already exist, so if it doesn't, you should first do a data.get() and then a data.set() if nothing exists
2. The docs show this example:

```js
let table = 'rain'
let key = `Wai'ale'ale`
let averageInches = 450

// Increment
await data.incr({table, key, averageInches})
```

This gives an error where it says "prop" is not defined.

I had to do this: 

```js
await data.incr({table, key, prop: 'averageInches'})
```

Is this correct? I see you passing 450 int the example, is that a way to give it an initial value? If so, how do I do it?

**Comment by brianleroux** [comment]
docs bug! thx for filing this / def need to pass the prop to increment

**Comment by pchinjr** [comment]
Should be resolved with #159 thanks!
